src/extension/background.js

```diff
@@ -12,9 +12,18 @@ const bgImages = require.context("../images/patterns/", false, /bg-.*\.svg/);
 const siteUrl = process.env.SITE_URL;
 
 const init = () => {
-  browser.browserAction.onClicked.addListener(() =>
-    browser.tabs.create({ url: siteUrl })
-  );
+  browser.browserAction.onClicked.addListener(() => {
+    browser.tabs.query({ currentWindow: true })
+      .then(tabs => {
+        const themerTab = tabs.find(tab => {
+          return tab.title === "Firefox Themer";
```

Comment:
Two suggestions here:
* I'd rather we searched on `tab.url === siteUrl` here. Someday we may translate this title into other languages and so this string comparison will no longer work
* This could be made more compact like so: `const themerTab = tabs.find(tab => tab.url === siteUrl);`

Reply:
@lmorchard I agree, comparing against the url would be better. Since `siteUrl` is just the base url, would it be better to do:
```js
const themerTab = tabs.find(tab => tab.url.includes(siteUrl));
```
since there could be a `?theme` query in the url?

Reply:
Oh, good catch! Yeah, `includes` is better than `=== ` - using `.indexOf(siteUrl) === 0` would be even more exact, though it's unlikely we'll find the siteUrl elsewhere in the url for a false positive